go.mod

```diff
@@ -1,9 +1,31 @@
-module github.com/gogama/incite
+module github.com/gogama/incite/v2
 
-go 1.14
+go 1.22
```

Comment:
Can we go any lower? I see the Go docs currently claim V2 SDK needs `go 1.21`, can we potentially go down there?